tests/testBackground.py

```diff
@@ -513,22 +458,8 @@ def testOnlyOneGridCell(self):
                 testval = afwMath.cast_BackgroundMI(backobj).getPixel(bctrl.getInterpStyle(), xpix, ypix)
                 self.assertAlmostEqual(testval/mean, 1)
 
-    def testTicket1681OffByOne(self):
```

Comment:
That's no different to what would happen in some other test if something broke prior to the assert being called so I don't see what the problem is.